frida-gum/src/symbol_util.rs

```diff
@@ -0,0 +1,69 @@
+use std::convert::TryInto;
+use std::fmt;
+use std::mem::MaybeUninit;
+use std::{ffi::CStr, str::Utf8Error};
+
+use frida_gum_sys as gum_sys;
+use gum_sys::{gum_symbol_details_from_address, GumDebugSymbolDetails};
+
+use crate::NativePointer;
+
+pub struct DebugSymbolDetails {
+    pub debug_symbol_details: GumDebugSymbolDetails,
```

Comment:
This should not be `pub`

Reply:
Fixed in 3eed93b8e9550de8ffc035f68c41369638eaf832